bookwyrm/tests/activitypub/test_base_activity.py

```diff
@@ -177,12 +177,21 @@ def test_to_model_many_to_many(self, *_):
                     "name": "gerald j. books",
                     "href": "http://book.com/book",
                 },
+                {
+                    "type": "Hashtag",
+                    "name": "#BookClub",
+                    "href": "http://example.com/tags/BookClub",
+                },
             ],
         )
         update_data.to_model(model=models.Status, instance=status)
         self.assertEqual(status.mention_users.first(), self.user)
         self.assertEqual(status.mention_books.first(), book)
 
+        hashtag = models.Hashtag.objects.filter(name="#BookClub").first()
+        self.assertIsNotNone(hashtag)
+        self.assertEqual(status.mention_hashtags.first(), hashtag)
```

Comment:
I don't have a nice local setup with two separate instances to manually test federation of statuses and the new hashtags, so I built the federation imports just based on these tests. Did I maybe miss something else?

Reply:
I have a pretty nice local setup to test federation now 🙌 if anybody else is interested, I wrote about how it works [here](https://amble.blog/2023/01/29/a-local-developent-setup-for-bookwyrm-federation/).